src/documentmanager/documentmanager.cpp

```diff
@@ -414,7 +418,7 @@ CodeEditor* DocumentManager::getCurrentDocument()
     // receive current sub wdw from area in focus
     // if current sub wdw is null - we return null to indicate search failure
     auto pCurrentDocument = pAreaInFocus->currentSubWindow();
-    return pCurrentDocument ? static_cast<CodeEditor*>(pCurrentDocument->widget()) : nullptr;
+    return pCurrentDocument ? qobject_cast<CodeEditor*>(pCurrentDocument->widget()) : nullptr;
```

Comment:
check nullptr again

Reply:
if qobject_cast fails & returns null_ptr then null_ptr is returned from method. Return value is checked by calling method.
Current method was designed that way

Reply:
see it, ok.